controllers/api/user-routes.js

```diff
@@ -44,6 +44,19 @@ router.post("/login", async (req, res) => {
 });
 
 // Sign-Up Endpoint(s)
+router.post('/', async (req, res) => {

```

Comment:
Please change `post('/',` to something more identifiable, like `post('/sign-up',` and update the `sign-up-handler`'s fetch request to match.